flow/metrics.yaml

```diff
@@ -69,3 +81,44 @@ components:
               x-field-uid: 1
             cut_through:
               x-field-uid: 2
+    Flow.TxRxReplication:
+      description: |-
+        Tx Rx replication factor is required to determine the ratio of expected number of Rx packets across all Rx ports
+        to the Tx packets for the configured flow.
+      type: object
+      properties:
+        choice:
+          type: string
+          default: one
+          x-field-uid: 1
+          x-enum:
+            one:
+              x-field-uid: 1
+            rx_ports_count:
```

Comment:
If my Rx is a `lag` with 2 ports, and I'm broadcasting, `rx_ports_count` would mean I shall expect twice more packets on that `lag` than `tx` value. But that is wrong. So we should have `rx_connection_count` instead to count `lag` as one.

Reply:
or just `rx_count`

Reply:
I have renamed it to just rx_count.